packages/commonwealth/client/scripts/hooks/useHandleInviteLink.ts

```diff
@@ -0,0 +1,84 @@
+import { useCallback, useEffect } from 'react';
+import { matchRoutes, useSearchParams } from 'react-router-dom';
+import { setLocalStorageRefcode } from '../helpers/localStorage';
+import app from '../state';
+import { useAuthModalStore } from '../state/ui/modals';
+import { useUserStore } from '../state/ui/user/user';
+import { AuthModalType } from '../views/modals/AuthModal';
+
+export const useHandleInviteLink = ({
+  isInsideCommunity,
```

Comment:
`isInsideCommunity` could be removed since we also check for `activeChainId` in this hook

Reply:
the main problem is that once the data is being loaded, isInsideCommunity is true but activeChainId is still undefined so I can't rely on it in this case.